overlord/devicestate/systems.go

```diff
@@ -426,3 +429,49 @@ func createSystemForModelFromValidatedSnaps(model *asserts.Model, label string,
 
 	return recoverySystemDir, nil
 }
+
+// ModelAndGadgetInfoFromSeed return the model assertion and gadget details
+// for the given system label.
+func (m *DeviceManager) ModelAndGadgetInfoFromSeed(wantedSystemLabel string) (*asserts.Model, *gadget.Info, error) {
+	if m.isClassicBoot {
```

Comment:
nitpicky code org wondering: Systems is defined in devicemgr.go, I wonder if it should move here or this move there?